packages/realm-tools/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@realm/tools",
+  "version": "0.0.1",
+  "description": "Tools useful for working with Realms",
+  "bin": {
+    "realm-schema": "./dist/realm-schema.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "lint": "eslint --ext .js,.ts .",
+    "prepare": "rm -rf dist && npm run build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/realm/realm-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/realm/realm-js/issues"
+  },
+  "keywords": [
+    "realm",
+    "tool"
+  ],
+  "author": {
+    "name": "Realm",
+    "email": "help@realm.io",
+    "url": "https://realm.io"
+  },
+  "contributors": [
+    "Kenneth Geisshirt <kenneth.geisshirt@mongodb.com> (https://github.com/kneth)"
+  ],
+  "license": "See the actual license in the file LICENSE",
+  "dependencies": {
+    "realm": ">=10",
+    "ts-command-line-args": "^2.2.1"
+  },
+  "devDependencies": {
+    "@types/command-line-args": "^5.2.0",
+    "@types/node": "^17.0.29"
```

Comment:
[Blocker] `typescript` needs to be added to `devDependencies`, otherwise anyone consuming this will be forced to install `typescript`.